.github/ISSUE_TEMPLATE.md

```diff
@@ -30,6 +34,9 @@ modinfo spl | grep -iw version
 
 ### Include any warning/errors/backtraces from the system logs
 <!-- 
+*IMPORTANT* - You should mark logs (see example below),
+otherwise Github will mess them up.
```

Comment:
`*IMPORTANT* - Please mark logs or Github will not display them correctly. An example is provided below.`?